src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static Task[] taskList = new Task[100];
+    static Boolean[] taskStatusList = new Boolean [100];
+    static int currentCount = 0;
+    static String dashedLine = "\t____________________________________________________________";
+
+    public static void markTask(int index){
+        taskStatusList[index - 1] = true;
+        String message = dashedLine + "\n" +
+                "\t Nice! I've marked this task as done: \n" +
+                "\t \t [X] " + taskList[index - 1].task + "\n" +
+                dashedLine ;
+        System.out.println(message);
+    }
+
+    public static void unMarkTask(int index){
```

Comment:
I think unmark is a word and a verb so you might want to change the name to unmarkedTask